src/db/migrations/20200110001514-create-user.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Users', {
```

Comment:
The table name should be singular and lowercase (another WeDevelop's "practice"). We can talk further on Slack.